suites/tipset/miner_flow.go

```diff
@@ -272,5 +268,6 @@ func TestMinerSubmitFallbackPoSt(t *testing.T, factory state.Factories) {
 
 	var powerSt power_spec.State
 	td.GetActorState(builtin_spec.StoragePowerActorAddr, &powerSt)
-	assert.Equal(t, drivers.InitialTotalNetworkPower+int64(32<<30), powerSt.TotalNetworkPower.Int64())
+	// FIXME: ensure this is the correct assertion to make after the changes that landed in specs-actors
```

Comment:
need to address before merging

Reply:
You want to inspect the TotalRawBytePower in power state, but then I think it should be still good.